bmp.h

```diff
@@ -1,13 +1,13 @@
 
-#include <cstdint.h>
+#include <cstdint>
 
 typedef uint8_t BYTE;
 typedef uint32_t DWORD;
-typedef int32_t LONG;
 typedef uint16_t WORD;
+typedef int32_t LONG;
+
 
-typedef struct
-{
+typedef struct{
```

Comment:
```suggestion
typedef struct {
```
Please add space after struct.